terraform/performance/main.tf

```diff
@@ -34,13 +34,15 @@ locals {
   validator_config        = "parameters.yml"
   final_validator_config  = "final_parameters.yml"
   cloudwatch_agent_config = "agent_config.json"
+  instance_temp_directory = "/tmp"
 }
 
 resource "local_file" "update-validation-config" {
-  content  = replace(replace(file("${var.test_dir}/${local.validator_config}"), 
+  content  = replace(replace(replace(replace(file("${var.test_dir}/${local.validator_config}"), 
```

Comment:
Is this correct?

Reply:
Yes. Its not a clean code but that's how we can replace the placeholder in a yaml file to make it more dynamically